GeoHealthCheck/static/lib/plotly/1.43.1/README.md

```diff
@@ -0,0 +1,19 @@
+# Plotly.js - Interactive Plots
```

Comment:
Do we have to keep plotly.js locally?  Can we pin and download at setup time?

Reply:
See https://gitter.im/geopython/GeoHealthCheck, yes is possible there is a stable CDN to download from at setup.